File: backend/pawnscan/src/main/java/edu/cit/garbo/pawnscan/features/auth/dto/GoogleAuthConfigResponse.java
package edu.cit.garbo.pawnscan.features.auth.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class GoogleAuthConfigResponse {

    private boolean configured;
    private String clientId;
}

File: mobile/app/src/main/java/com/cit/pawnscan/features/auth/api/GoogleAuthConfigResponse.kt
package com.cit.pawnscan.features.auth.api

import com.google.gson.annotations.SerializedName

data class GoogleAuthConfigResponse(
    @SerializedName("configured")
    val configured: Boolean = false,

    @SerializedName("clientId")
    val clientId: String? = null
)
